docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       BOULDER_CONFIG_DIR: test/config
       GOCACHE: /boulder/.gocache/go-build
       GOFLAGS: -mod=vendor
+      GOMAXPROCS: 2
```

Comment:
Do you intend this to be in the PR, so that it affects our local runs as well?

Reply:
Nope!